ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Byron.hs

```diff
@@ -742,14 +743,19 @@ instance (ByronGiven, Typeable cfg, ConfigContainsGenesis cfg)
   --
   -- TODO #514: This is still missing the other cases (this shouldn't be a
   -- newtype)
-  data GenTx (ByronBlockOrEBB cfg) = ByronTx { unByronTx :: CC.UTxO.ATxAux ByteString }
+  data GenTx (ByronBlockOrEBB cfg) = ByronTx
+    { byronTx   :: CC.UTxO.ATxAux ByteString
```

Comment:
Perhaps this one should have been strict though. I don't know if there was a good reason for it not being a newtype previously.